assets/controllers/qrcode_controller.js

```diff
@@ -0,0 +1,10 @@
+import { Controller } from '@hotwired/stimulus';
+import jrQrcode from 'jr-qrcode';
+
+export default class extends Controller {
+  static values = { url: String }
```

Comment:
So there is magic somewhere where `this.values.url` is being converted into `this.urlValue`?

Reply:
Yes, stimulus "feature"